tests/e2e/test_api.py

```diff
@@ -659,25 +659,35 @@ def test_forbidden_user():
     assert response.status_code == requests.codes.forbidden
 
 
-def test_feedback() -> None:
-    """Check if feedback is properly stored.
+@pytest.mark.cluster()
+def test_feedback_can_post_with_wrong_token():
+    """Test posting feedback with improper auth. token."""
+    response = client.post(
+        "/v1/feedback",
+        json={
+            "conversation_id": CONVERSATION_ID,
+            "user_question": "what is OCP4?",
+            "llm_response": "Openshift 4 is ...",
+            "sentiment": 1,
+        },
+        timeout=BASIC_ENDPOINTS_TIMEOUT,
+        headers={"Authorization": "Bearer wrong-token"},
+    )
+    assert response.status_code == requests.codes.forbidden
 
-    This is a full end-to-end scenario where the feedback is stored,
-    retrieved and removed at the end (to avoid leftovers).
-    """
-    # check if feedback is enabled
-    response = client.get("/v1/feedback/status", timeout=BASIC_ENDPOINTS_TIMEOUT)
-    assert response.status_code == requests.codes.ok
-    assert response.json()["status"]["enabled"] is True
 
-    # check the feedback store is empty
-    empty_feedback = client.get("/v1/feedback/list", timeout=BASIC_ENDPOINTS_TIMEOUT)
-    assert empty_feedback.status_code == requests.codes.ok
-    assert "feedbacks" in empty_feedback.json()
-    assert len(empty_feedback.json()["feedbacks"]) == 0
+@pytest.mark.standalone()
+def test_feedback_storing__standalone():
```

Comment:
is double `__` there on purpose?

Reply:
yes

Reply:
and why?

Reply:
To separate 
```
test_feedback_storing__standalone
test_feedback_storing__cluster
```
You don't like it?:)

Reply:
that's ok, just not used much as it has no meaning